Pipfile

```diff
@@ -37,6 +37,7 @@ sphinx = ">=1.8,<2"
 [packages]  # Make sure to keep in sync with setup.py requirements.
 arrow = ">=0.15.0,<1"
 funcy = ">=1.7.3,<2"
+dataclasses = ">=0.6"
```

Comment:
Per [PEP508](https://www.python.org/dev/peps/pep-0508/), I think we may actually want `>=0.7,<1; python_version < 3.7` here, though I'm not sure how that will manifest itself in the lockfile. The real longer-term answer we might want down the road might be to switch to `poetry` which seems a bit better suited to package development.

Reply:
Unfortunately, it looks like pipenv explicitly doesn't support multiple versions so that syntax fails:
https://github.com/pypa/pipenv/issues/1050

I believe the workaround is to not specify that Python version in the Pipfile. I got around this for now by specifying the version as 0.6, which is compatible with Python 3.8, though I agree 0.7 would be preference.